var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -52,9 +52,11 @@ class Metis(Package):
     conflicts('@:4.999', when='+real64')
 
     depends_on('cmake@2.8:', when='@5:', type='build')
+    depends_on('cmake@3:', when='platform=darwin')
```

Comment:
```suggestion
    depends_on('cmake@3.2:', when='@5: platform=darwin', type='build')
```

Reply:
I didn't know the syntax was forcing this. Even better solution.